util/setchplenv.bash

```diff
@@ -38,11 +38,10 @@ fi
 export CHPL_HOME=$chpl_home
 echo "Setting CHPL_HOME to $CHPL_HOME"
 
-export CHPL_HOST_PLATFORM=`"$CHPL_HOME"/util/chplenv/chpl_platform.py`
-echo "Setting CHPL_HOST_PLATFORM to $CHPL_HOST_PLATFORM"
+CHPL_BIN_SUBDIR=`"$CHPL_HOME"/util/chplenv/chpl_bin_subdir.py`
 
-export PATH="$CHPL_HOME"/bin/$CHPL_HOST_PLATFORM:"$CHPL_HOME"/util:"$MYPATH"
-echo "Updating PATH to include $CHPL_HOME/bin/$CHPL_HOST_PLATFORM"
+export PATH="$CHPL_HOME"/bin/$CHPL_BIN_SUBDIR:"$CHPL_HOME"/util:"$MYPATH"
+echo "Updating PATH to include $CHPL_HOME/bin/$CHPL_BIN_SUBDIR"
 echo "                     and $CHPL_HOME/util"
```

Comment:
Do we have to change non bash ones too?

Reply:
Yes but I wanted to get feedback on the approach before changing *everything*